### ISSUE TITLE
No entrypoint/cmd found

### ISSUE THREAD

**Issue by strarsis** [bylexus/docker-apache-php53#2]
I cloned this repository and build the image from the Dockerfile locally, 
when I try to run it I get the following error:

```
no such file or directory
docker: Error response from daemon: Container command not found or does not exist..
```

**Comment by bylexus** [comment]
Hi,

I cannot find any problems with the image. I just issued the following command:

`docker run -d -P bylexus/apache-php53`

which runs without any issues.

Maybe this is related to issue #1 (documentation issue)?
I fixed the README doc in commit 397e79a858969ef92a802850d5cac4e964df3cec.

**Comment by strarsis** [comment]
Thanks, the fixed image name and the command argument fixed the issue.
